src/api.ts

```diff
@@ -232,3 +232,16 @@ export async function ensureDependencyInstalled(
 
   await addDependency(name, resolvedOptions);
 }
+
+export async function dedupeDependencies(options: OperationOptions = {}) {
+  const resolvedOptions = await resolveOperationOptions(options);
+  if (["bun", "deno"].includes(resolvedOptions.packageManager?.name ?? "")) {
+    throw new Error(
+      `Deduping is currently not supported for \`${resolvedOptions.packageManager?.name ?? "unknown package manager"}\``,
```

Comment:
In this case, we can fallback to:

- Remove known lockfiles
- Run install

Reply:
> In this case, we can fallback to:
> 
> * Remove known lockfiles
> * Run install

Should I use another utility to remove the lockfile or do it in place?

Reply:
Inline is fine. We can support `{ recreateLockFile?: boolean }` option also to force this behavior optionaly.

Reply:
> Inline is fine. We can support `{ recreateLockFile?: boolean }` option also to force this behavior optionaly.

I have tried it so that you can also pass a false for deno or bun. This should ensure that you can also prevent recreating. Might be nice for nuxi.